src/components/Signup/Signup.js

```diff
@@ -234,6 +237,7 @@ const Signup = function ({
             onSubmit={onSubmit}
             validate={validate}
           >
+            <input type='hidden' id="zc_gad" name="zc_gad"/>
```

Comment:
How is this field used?

Reply:
This PR was brought into life again only to get hold of a cdn link with the script and hidden field to show to zoho, just in case they answer our question. And that's all. https://cdn.fromdoppler.com/doppler-webapp/build3180/#/signup?gclid=dfsflksdjfslkdfjs

Reply:
Thanks for the clarification!